deployer skip_symlinks (#3335)
conan-io/docs: reference/extensions/deployers.rst

  See :ref:`the Conan stability<stability>` section for more information.

configuration
^^^^^^^^^^^^^

> Both the ``full_deploy`` and the ``direct_deploy`` can define the conf ``tools.deployer:symlinks`` to ``False``

Reviewer: ```suggestion
Both the ``full_deploy`` and the ``direct_deploy`` understand when the conf ``tools.deployer:symlinks`` is set to ``False``
```
